docs/rules/no-confusing-set-timeout.md

```diff
@@ -0,0 +1,63 @@
+# Disallow using confusing setTimeout in test (`no-confusing-set-timeout`)
+
+<!-- end auto-generated rule header -->
+
+`jest.setTimeout` can be called multiple times anywhere within a single test
+file. However, only the last call will have an effect, and it will actually be
+invoked before any other jest functions.
+
+## Rule details
+
+This rule describes some tricky ways about `jest.setTimeout` that should not
```

Comment:
```suggestion
This rule describes some tricky ways about `jest.setTimeout` that is not
```

Reply:
fwiw this should actually be "are" since its multiple; but I've suggested a bigger rewrite to this below.